lib/jobs/install-os.js

```diff
@@ -96,6 +96,23 @@ function installOsJobFactory(
                 }
             });
         }
+
+        if (this.options.installPartitions) {
+            _.forEach(this.options.installPartitions, function(partition) {
+                assert.string(partition.mountPoint);
+                assert.string(partition.size);
```

Comment:
in my opinion. I don't think so. I don't want to let users or the document have more understanding for when  'size' is omitted. it's not straightforward.  I also don't think we provide both string/number at the same time. we have to understand more about the input.  it's not straightforward for quickly understanding.  either number or string, choose one. actually, I still have concerns about why provide more choices for 'installDisk' for this default sequences. I have to read more than 3 times to understand what it's happening.